tests/integration_tests/functional/test_rate_limiter.py

```diff
@@ -178,6 +177,13 @@ def test_rx_rate_limiting_cpu_load(test_microvm_with_api, network_config):
     )
     _iperf_out = _run_local_iperf(iperf_cmd)
 
+    # Stop the CPU monitor thread - otherwise it will crash after the microvm goes away due to no firecracker process
+    # existing anymore
+    test_microvm.cpu_load_monitor.signal_stop()
```

Comment:
should we call test_microvm.kill() here instead because the kill() is calling signal_stop/join/check_samples on the cpu_load_monitor as shown here:
https://github.com/firecracker-microvm/firecracker/pull/3442/files#diff-d8abdd84a323dd0369671d7aa8e0359578e641946c5b764a8b7c82cb36167486L217

Reply:
See https://github.com/firecracker-microvm/firecracker/pull/3442#discussion_r1108174457